Formula/git-gui.rb

```diff
@@ -0,0 +1,49 @@
+class GitGui < Formula
+  desc "Tcl/Tk UI for the git revision control system"
+  homepage "#{Formula["git"].homepage}"
+  url "#{Formula["git"].stable.url}"
+  sha256 "#{Formula["git"].stable.checksum}"
+  head "#{Formula["git"].head}", :shallow => false
+
+  depends_on "git"
+  depends_on "tcl-tk"
+
+  def install
+    ENV["V"] = "1" # build verbosely
+    args = %W[
+      prefix=#{prefix}
+      gitexecdir=#{libexec}/git-core
+      sysconfdir=#{etc}
+      CC=#{ENV.cc}
+      CFLAGS=#{ENV.cflags}
+      LDFLAGS=#{ENV.ldflags}
+    ]
+
+    # By setting TKFRAMEWORK to a non-existent directory we ensure that
+    # the git makefiles don't install a .app for git-gui
+    # We also tell git to use the homebrew-installed wish binary from tcl-tk.
+    # See https://github.com/Homebrew/homebrew-core/issues/36390
+    tcl_bin = Formula["tcl-tk"].opt_bin
+    args += %W[
```

Comment:
Please make these part of the `args` array above as they are not e.g. dynamically assigned based on a conditional.